x-pack/plugin/sql/qa/server/src/main/java/org/elasticsearch/xpack/sql/qa/rest/BaseRestSqlTestCase.java

```diff
@@ -210,6 +219,24 @@ protected static void deleteIndex(String name) throws IOException {
         deleteIndex(provisioningClient(), name);
     }
 
+    protected void createDataStream(String dataStreamName) throws IOException {
```

Comment:
I see this is duplicated in BaseRestSqlTestCase and JdbcIntegrationTestCase, can't we move it to a superclass?

Reply:
Makes sense, I've just imported them.